src/components/Popover/index.tsx

```diff
@@ -81,13 +81,16 @@ export interface PopoverProps {
   offsetY?: number
   hideArrow?: boolean
   showInline?: boolean
+  // time delay in milliseconds before the popover shows
```

Comment:
This should be JSDoc:

```suggestion
  /** time delay in milliseconds before the popover shows */
```

Reply:
deleted the delay